politeiawww/api/v1/v1.go

```diff
@@ -219,6 +231,17 @@ var (
 		PropVoteStatusFinished:    "voting finished",
 		PropVoteStatusDoesntExist: "proposal does not exist",
 	}
+
+	// NotificationIdentifier converts a Notification Id to human readable text
+	NotificationIndentifier = map[NotificationType]string{
+		NotTypeSignupPaywallPaymentConfirmed: "Paywall payment for Politeia access confirmed",
```

Comment:
This is very nitpicky but can you make the strings lowercase to be consistent with the others?